src/FINDOLOGIC/Export/Exporter.php

```diff
@@ -63,10 +63,11 @@ protected function __construct($itemsPerPage)
      * @param array $items Array of items to serialize. All of them are serialized, regardless of $start and $total.
      * @param int $start Assuming that $items is a fragment of the total, this is the global index of the first item in
      *      $items.
+     * @param int $count The number of items processed per export step.
```

Comment:
Given the peculiar meaning of this value we discussed, the documentation should mention that this is the number of expected items, and does not necessarily have to match the actual number in case of problems for individual items.

Reply:
Possible text: The number of items requested for this export step. Actual number of items can be smaller due to errors, and can not be greater than the requested count, because that would indicate that the requested count is ignored when generating items.